### ISSUE TITLE
Abuse of HTML5 elements causes improper Aria Landmarks

### ISSUE THREAD

**Issue by avaaec** [ManifoldScholar/manifold#2262]
This is an issue that appears in multiple places around manifold, much like global issues around groupings, headings, and single item lists. Collectively, small but recurring issues like this have a significant negative impact on accessibility. I will focus on one specific instance of this issue.
## Steps to Reproduce
1. Go to edge.manifoldapp.org/projects/a-study-in-scarlet
2. Notice that there are three `<header>` elements in the main region of the page
3. Using a tool like a screen reader or accessibility browser extension like Ainspecter, WAVE Accessibility to see that this causes there to be three Aria Landmarks with the role "Banner"

## Observed Behavior
Using HTML5 elements like `<nav>` or `<header>` with out semantic reason creates unnecessary and improper Aria Landmarks that are confusing to people who rely on assistive technologies that rely on landmarks to navigate a page.

## Expected Behavior 
Only use these semantic elements in places where a landmark belongs and use them alongside traditional aria `role=` properties. You can learn more about this [here](https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html).

## Testing Environment
NVDA 2019.1.1
ESR Firefox 60.7.0esr

**Comment by avaaec** [comment]
When writing this issue I noticed that a lot of places where I had seen this issue previously had been fixed. :) Nice!

**Comment by zdavis** [comment]
We believe we've addressed this problem now—that said, if it pops up again, please feel free to reopen this issue!

**Comment by avaaec** [comment]
This problem appears in: 
* The user settings panel of the site banner
* Reader -> banner Menu Panel
* Reader -> Table of contents panel

There should not be landmarks inside of popup sub menus or panels like this. Landmarks should be static unless a major portion of the page is changing. Content that is usually hidden inside of an organization control like a menu should not have landmarks.

**Comment by Santoz2** [comment]
In addition to what Ava mentioned, there is another odd panel landmark in the new Adjust Appearance panel:

The title of the heading for the panel is given a Header tag, making it a Banner landmark.
